demisto_sdk/commands/common/content/objects/pack_objects/pack.py

```diff
@@ -117,16 +117,28 @@ def integrations(self) -> Iterator[Integration]:
         return self._content_files_list_generator_factory(dir_name=INTEGRATIONS_DIR,
                                                           suffix="yml")
 
+    @property
+    def integrations_amount(self):
+        return len([integration for integration in self.integrations])
+
     @property
     def scripts(self) -> Iterator[Script]:
         return self._content_files_list_generator_factory(dir_name=SCRIPTS_DIR,
                                                           suffix="yml")
 
+    @property
+    def scripts_amount(self):
+        return len([script for script in self.scripts])
+
     @property
     def playbooks(self) -> Iterator[Playbook]:
         return self._content_files_list_generator_factory(dir_name=PLAYBOOKS_DIR,
                                                           suffix="yml")
 
+    @property
+    def playbooks_amount(self):
+        return len([playbook for playbook in self.playbooks])
```

Comment:
unfortunately your suggestion won't work because self.playbooks returns a generator object, in my case i want to know the amount of the content entities. when doing len on a generator object you will get ```TypeError```

```
Traceback (most recent call last):
  File "/Applications/PyCharm CE.app/Contents/plugins/python-ce/helpers/pydev/_pydevd_bundle/pydevd_exec2.py", line 3, in Exec
    exec(exp, global_vars, local_vars)
  File "<input>", line 1, in <module>
TypeError: object of type 'generator' has no len()
```

Reply:
`len(tuple(self.playbooks))`?

Reply:
that would work

Reply:
done